database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -17,6 +17,7 @@ public function up()
             $table->bigIncrements('id');
```

Comment:
Where is usage of https://docs.google.com/spreadsheets/d/1bqeqmlFx-0867eX4onaM6IZYLpkewONk3wI8i1vICTs/edit#gid=605672571 Logging out

Reply:
feature logout has used in ruotes/web ``` Auth::routes() ```. 
it has written in file Illuminate\Foundation\Auth\AuthenticatesUsers.